client-v2/src/test/java/com/clickhouse/client/api/data_formats/internal/BinaryStreamReaderTests.java

```diff
@@ -0,0 +1,29 @@
+package com.clickhouse.client.api.data_formats.internal;
+
+import com.clickhouse.data.ClickHouseColumn;
+import com.clickhouse.data.ClickHouseDataType;
+import com.clickhouse.data.ClickHouseInputStream;
+import com.clickhouse.data.format.BinaryStreamUtils;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+import java.io.ByteArrayInputStream;
+import java.io.ByteArrayOutputStream;
+import java.time.LocalDate;
+import java.util.TimeZone;
+
+public class BinaryStreamReaderTests {
+
+
+    @Test(groups = {"unit"}, enabled = false)

```

Comment:
We need to extend testing to all our types

Reply:
I will do it right after we are done with this PR.